<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainBox</class>
 <widget class="QWidget" name="MainBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>975</width>
    <height>236</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="multimeter_frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <property name="leftMargin">
         <number>8</number>
        </property>
        <property name="topMargin">
         <number>8</number>
        </property>
        <property name="rightMargin">
         <number>8</number>
        </property>
        <property name="bottomMargin">
         <number>8</number>
        </property>
        <item>
         <widget class="QPushButton" name="btn_fibd_device">
          <property name="text">
           <string>Поиск мультиметра</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Адрес:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sb_address"/>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>timeout:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sb_timeout"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QPushButton" name="btn_power">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>33</width>
              <height>70</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton#btn_power { 
            image: url(:/power_knock/image/black/off.png); 
            border-style: inset; 
} 
            QPushButton#btn_power:checked { 
            image: url(:/power_knock/image/green/on.png); 
            border-style: inset; 
}</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QPushButton" name="btn_1">
            <property name="text">
             <string>Auto</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_2">
            <property name="text">
             <string>--&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_3">
            <property name="text">
             <string>&lt;--</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_4">
            <property name="text">
             <string>Shift</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="leftMargin">
           <number>8</number>
          </property>
          <property name="topMargin">
           <number>8</number>
          </property>
          <property name="rightMargin">
           <number>8</number>
          </property>
          <property name="bottomMargin">
           <number>8</number>
          </property>
          <item>
           <widget class="QLabel" name="multimeter_display">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>512</width>
              <height>64</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>48</pointsize>
             </font>
            </property>
            <property name="text">
             <string>---</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="btn_5">
              <property name="text">
               <string>V</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_6">
              <property name="text">
               <string>I</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_7">
              <property name="text">
               <string>~V</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_8">
              <property name="text">
               <string>~I</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_9">
              <property name="text">
               <string>R</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_10">
              <property name="text">
               <string>L</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_11">
              <property name="text">
               <string>C</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_12">
              <property name="text">
               <string>F</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QPushButton" name="btn_16">
            <property name="text">
             <string>Comp</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_15">
            <property name="text">
             <string>^</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_14">
            <property name="text">
             <string>v</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_13">
            <property name="text">
             <string>Flt</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="grapher_layout"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
